File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/parking/FindParkingPayloadResponse.java
package com.lattis.ellipse.data.network.model.response.parking;

import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by ssd3 on 4/5/17.
 */

public class FindParkingPayloadResponse {

    @SerializedName("parking_spots")
    List<FindParkingDataResponse> findParkingDataResponses;

    List<FindParkingDataResponse> findParkingDataResponses(){
        return findParkingDataResponses;
    }
}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/response/parking/FindParkingPayloadResponse.kt
package com.lattis.data.entity.response.parking

import com.google.gson.annotations.SerializedName

/**
 * Created by ssd3 on 4/5/17.
 */
class FindParkingPayloadResponse {
    @SerializedName("parking_spots")
    var findParkingDataResponses: List<FindParkingDataResponse>? = null
    fun findParkingDataResponses(): List<FindParkingDataResponse>? {
        return findParkingDataResponses
    }
}